tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -173,7 +173,9 @@ Spawn Notebook With Arguments  # robocop: disable
          Exit For Loop If  ${spawn_fail} == False
          Click Element  xpath://span[@id='jupyterhub-logo']
       ELSE
+         Run Keyword If  ${index} == ${retries}  Fail
          Click Element  xpath://span[@id='jupyterhub-logo']
+         Sleep  ${retries_delay}
```

Comment:
Sure, Your proposal looks good to me. Thanks @lugi0

Reply:
I just realized the PR was sent by @kpouget and @tarukumar only added the reviewers. Sorry and thanks for the contribution @kpouget :D

Reply:
>  I think the expectation is that the spawner would load everything rather quickly regardless of user count, and adding an additional artificial wait time would hide the issue that these FAILs would show.

the reason why we want this delay is to have a chance that the test succeeds, even if it takes time.
Currently we're at 80% of failures (out of 100 users) because of this bottleneck, we need to get passed this quickly.
The delay can be 0 by default to keep the existing behavior.